src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSparkUnitTest.java

```diff
@@ -58,10 +60,8 @@ private static void setup() throws IOException{
         rawFASTQFiles = new ArrayList<>();
         expectedAssembledFASTAFiles = new HashMap<>();
 
-        final File original_test_data_dir = new File(getTestDataDir(), "spark/sv/RunSGAViaProcessBuilderOnSpark/");
-        TEST_DATA_DIR = Files.createTempDirectory( "whatever" ).toAbsolutePath().toFile();
-        TEST_DATA_DIR.deleteOnExit();
-        FileUtils.copyDirectory(original_test_data_dir, TEST_DATA_DIR);
+        TEST_DATA_DIR = new File(getTestDataDir(), "spark/sv/RunSGAViaProcessBuilderOnSpark/");
+        tempWorkingDir = BaseTest.createTempDir("whatever");
```

Comment:
I think you don't need this `tempWorkingDir` to be an instance variable on the test object; it's only used in the `setup` method.